ports/zephyr/boards/nucleo_h743zi.conf

```diff
@@ -0,0 +1,10 @@
+# disable console subsys to get REPL working
+CONFIG_CONSOLE_SUBSYS=n
+
+# flash and file system config
+CONFIG_FLASH=y
+CONFIG_FLASH_MAP=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+
+CONFIG_FILE_SYSTEM=y
+CONFIG_FILE_SYSTEM_LITTLEFS=y
```

Comment:
This is probably not what you want, unless you intend to use LittleFS under the hood in C without exposing it to MicroPython scripts. Instead you need to make sure that Makefile options `MICROPY_VFS_LFS1` or `MICROPY_VFS_LFS2` are set. The latter is set by default in the Zephyr port. See https://github.com/micropython/micropython/blob/master/ports/zephyr/Makefile#L19-L21

Reply:
That's right, I didn't really see that. Thanks for the input! I have adjusted the config file accordingly. Using the make process as following, the filesystem is working:

`make BOARD=nucleo_h743zi MICROPY_VFS_LFS2=1  flash`

Reply:
The net result looks fine, but I recommend that you squash your cleanup commit into your original commit

Reply:
Good idea. Tnx!